lib/commands.js

```diff
@@ -61,8 +57,12 @@ commands.list.forEach(function (command) {
             }
             return this.internal_send_command(new Command(command, arr, callback));
         };
+        //alias commands with illegal function names (e.g. NR.RUN becomes NR_RUN and nr_run)
+        if (commandName !== command) {
+            RedisClient.prototype[commandName.toUpperCase()] = RedisClient.prototype[commandName] = RedisClient.prototype[command];
+        }
```

Comment:
I've read through your request a few times and I can't square what you're trying to say aside from reverting and removing the entire `if` block. However, removing the `if (commandName !== command) { ... }` block renders the code unusable. For example:
```js
redis.addCommand('ft.info');
client = redis.createClient(require('../localredis.json'));
console.log(client.multi()['FT.INFO']); //-> [Function: ft_info]
console.log(client.multi()['ft.info']); //-> [Function: ft_info]
console.log(client.multi().ft_info) //-> undefined
```

*All* modules are supposed to have a dot namespacing, so this is rather big deal. No one will want to use bracket notation.